cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -131,6 +134,43 @@ def test_embedded_ansible_repository_crud(ansible_repository, wait_for_ansible):
     assert view.description.value == updated_description
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_repository_branch_crud(appliance, request, wait_for_ansible):
+    """
+    Ability to add repo with branch (without SCM credentials).
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/12h
+        tags: ansible_embed
+    """
+    repositories = appliance.collections.ansible_repositories
+    try:
+        playbooks_yaml = cfme_data.ansible_links.playbook_repositories
+        playbook_name = getattr(request, 'param', 'embedded_ansible')
+        repository = repositories.create(
+            name=fauxfactory.gen_alpha(),
+            url=getattr(playbooks_yaml, playbook_name),
+            description=fauxfactory.gen_alpha(),
+            scm_branch="second_playbook_branch"
+        )
+    except (KeyError, AttributeError):
+        message = "Missing ansible_links content in cfme_data, cannot setup repository"
+        logger.exception(message)  # log the exception for debug of the missing content
+        pytest.skip(message)
+    view = navigate_to(repository, "Details")
+    assert (
+        view.entities.summary("Repository Options").get_text_of("SCM Branch")
+        == repository.scm_branch
+    )
```

Comment:
I'd ask you to add appropriate assert message and you can easily make this statement more readable by assiging view field value to variable 

`scm_branch = view.entities.summary("Repository Options").get_text_of("SCM Branch")`

Reply:
Okay I will update.